portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/cpdocumentsandmedia/CPDocumentsandmedia.testcase

```diff
@@ -55,47 +55,10 @@
 		</execute>
 	</command>
 
-	<command name="AddDocument" priority="4">
```

Comment:
@brianwulbern are we taking out these two tests in `CPDocumentsandmedia.testcase`? It looks like `AddDocument` and `AddFolder` are getting knocked out. is this intentional?